app/backend/routes/user/types.ts

```diff
@@ -0,0 +1,36 @@
+import type { ServerWebSocket } from "bun";
+import { z } from "zod";
+
+// * Set of rules for the profile
+// * Defines how the data should be validated
+// * Set what are valid inputs and what fields are allowed
+
+export type ChatSocket = ServerWebSocket<{ user: string }>;
```

Comment:
This line is still redundant see in app/backend/lib/types.ts

Reply:
Ah yeah i didnt see it, i removed it! Now should be better!